tests/examples/hello/hello/hello.py

```diff
@@ -0,0 +1,167 @@
+    """Create a Legate to task to print a message
+
+    Args:
+        message (str): The message to print
+    """
+    task = user_context.create_auto_task(HelloOpCode.HELLO_WORLD)
+    task.add_scalar_arg(message, types.string)
+    task.execute()
+
+
+def print_hellos(message: str, n: int) -> None:
+    """Create a Legate to task launch to print a message
+       in n replicas of the task
+
+    Args:
+        message (str): The message to print
+        n (int): The number of times to print
+    """
+    launch_domain = Rect(lo=[0], hi=[n], exclusive=True)
+    task = user_context.create_manual_task(
+        HelloOpCode.HELLO_WORLD, launch_domain=launch_domain
+    )
+    task.add_scalar_arg(message, types.string)
+    task.execute()
+
+
+def _get_legate_store(input: Any) -> Store:
+    """Extracts a Legate store from any object
+       implementing the legete data interface
+
+    Args:
+        input (Any): The input object
+
+    Returns:
+        Store: The extracted Legate store
+    """
+    if isinstance(input, Store):
+        return input
+    data = input.__legate_data_interface__["data"]
+    field = next(iter(data))
+    array = data[field]
+    _, store = array.stores()
+    return store
+
+
+def to_scalar(input: Store) -> float:
+    """Extracts a Python scalar value from a Legate store
+       encapsulating a single scalar
+
+    Args:
+        input (Store): The Legate store encapsulating a scalar
+
+    Returns:
+        float: A Python scalar
+    """
+    buf = input.storage.get_buffer(np.float32().itemsize)
+    result = np.frombuffer(buf, dtype=np.float32, count=1)
+    return float(result[0])
+
+
+def zero() -> Store:
+    """Create a Legates store representing a single zero scalar
+
+    Returns:
+        Store: A Legate store representing a scalar zero
+    """
+    data = bytearray(4)
+    buf = struct.pack(f"{len(data)}s", data)
+    future = get_legate_runtime().create_future(buf, len(buf))
+    return user_context.create_store(
+        types.float32,
+        shape=(1,),
+        storage=future,
+        optimize_scalar=True,
+    )
+
+
+def iota(size: int) -> Store:
+    """Enqueues a task that will generate a 1-D array
+       1,2,...size.
+
+    Args:
+        size (int): The number of elements to generate
+
+    Returns:
+        Store: The Legate store that will hold the iota values
+    """
+    output = user_context.create_store(
+        types.float32,
+        shape=(size,),
+        optimize_scalar=True,
+    )
+    task = user_context.create_auto_task(
+        HelloOpCode.IOTA,
+    )
+    task.add_output(output)
+    task.execute()
+    return output
+
+
+def sum(input: Any) -> Store:
+    """Sums a 1-D array into a single scalar
+
+    Args:
+        input (Any): A Legate store or any object implementing
+                     the Legate data interface.
+
+    Returns:
+        Store: A Legate store encapsulating the array sum
+    """
+    input = _get_legate_store(input)
```

Comment:
mypy might not like this, in which case it will want you to name the "sanitized" variable something different.

Reply:
mypy in general chokes on the examples for a variety of reasons. I'm just excluding them for now. But I also renamed this `store_input` for this example.